### DISCUSSION TITLE
500 internal server error while accessing hx-requests in Django project

### DISCUSSION THREAD

**Discussion by SKAv81** [bigskysoftware/htmx#1219] [General]
I am working on a Django project to create a dynamic form using HTMX using a YouTube video.. As I am clicking "Add" button to add a form, no form gets added.. and as I inspect the browser console, I get the following error:

GET http://127.0.0.1:8000/businesscase/htmx/project-form/ 500 (Internal Server Error) lr @ htmx.org@1.8.5:1 (anonymous) @ htmx.org@1.8.5:1 i @ htmx.org@1.8.5:1

Response Status Error Code 500 from /businesscase/htmx/project-form/ St @ htmx.org@1.8.5:1 ee @ htmx.org@1.8.5:1 Q @ htmx.org@1.8.5:1 fr @ htmx.org@1.8.5:1 o.onload @ htmx.org@1.8.5:1 load (async) lr @ htmx.org@1.8.5:1 (anonymous) @ htmx.org@1.8.5:1 i @ htmx.org@1.8.5:1

I tried to search a lot in Google to get rid of this error, but unfortunately all I received was to refresh browser, clear cache etc which I did but the problem persists..

I'd really appreciate of any quick solution here.. many thanks..

**Comment by SciStarter** [comment]
The problem is in your server code, and so it's not related to HTMX or anything that's happening in your browser. I would suggest looking at your terminal where you're running your server, and look for error messages there.
